api/src/main/java/javax/mvc/engine/ViewEngineContext.java

```diff
@@ -74,21 +71,6 @@
     Models getModels();
 
     /**
-     * Returns HTTP request object from the Servlet container.
-     *
-     * @return HTTP request object.
-     */
-    HttpServletRequest getRequest();
-
-    /**
-     * Returns HTTP response object from the servlet container. The underlying
-     * output stream should be used to write the result of processing a view.
-     *
-     * @return HTTP response object.
-     */
-    HttpServletResponse getResponse();
-
```

Comment:
If we remove the `HttpServletResponse` shouldn't we then add something like an `OutputStream`? Currently`ViewEngine#processView` states:

> Processing a view involves merging the model and template data and writing the result to an output stream.

Should the implementer decide to which stream he wants to write?